pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -1388,7 +1388,7 @@ func runCDCFineGrainedCheckpointingBenchmark(
 			return fmt.Errorf("expected %d, got %d", expected, unique)
 		}
 
-		if dupes > dupeLimit {
+		if dupeLimit > 0 && dupes > dupeLimit {
```

Comment:
If we aren't going to use `dupeLimit`, should we just delete all the related code?